dashboard/src/components/DeploymentForm/DeploymentForm.tsx

```diff
@@ -52,11 +57,20 @@ export default function DeploymentForm() {
   const [releaseName, setReleaseName] = useState("");
   const [appValues, setAppValues] = useState(selectedPackage.values || "");
   const [valuesModified, setValuesModified] = useState(false);
+  const [serviceAccountList, setServiceAccountList] = useState([] as string[]);
+  const [reconciliationOptions, setReconciliationOptions] = useState({} as ReconciliationOptions);
 
   const error = apps.error || selectedPackage.error;
 
   const [pluginObj] = useState({ name: pluginName, version: pluginVersion } as Plugin);
 
+  const onChangeSA = (e: React.FormEvent<HTMLSelectElement>) => {
+    setReconciliationOptions({
+      ...reconciliationOptions,
+      serviceAccountName: e.currentTarget.value,
```

Comment:
In the future, we may want to allow modifying other parameters, like the reconciliation interval, that's why I'm storing this obj instead of just the SA

Reply:
Yep, sound good.